static/app/views/issueList/overview.tsx

```diff
@@ -156,6 +158,8 @@ class IssueListOverview extends React.Component<Props, State> {
 
     return {
       groupIds: [],
+      reviewedIds: [],
+      forReview: false,
```

Comment:
can we remove these from state since they aren't used in render? This would also change how we're calling this.fetchData, would need to call ourselves